### PR TITLE
tau-helium: init at 1.5.8

### PR DESCRIPTION
**Pull request by Ashvith10**
## Description of changes
Adds tau-helium GTK theme.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Ashvith10** [comment]
Will have to wait for #251846 to complete, as the maintainer name wasn't added in this particular branch.

File: pkgs/data/themes/tau-helium/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  mesonFlags = [
+    "-Dgtk4=true"
+    "-Dshell=false"
+  ];
+
+  meta = with lib; {
+    description = "The GTK/GNOME Shell theme for tauOS";
+    homepage = "https://github.com/tau-OS/tau-helium";
+    license = licenses.gpl3;
```
Comment (Lord-Valen) [inline pkgs/data/themes/tau-helium/default.nix:35]:
```suggestion
    license = licenses.gpl3Only;
```

File: pkgs/data/themes/tau-helium/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tau-helium";
+  version = "1.5.5";
+
+  src = fetchFromGitHub {
+    owner = "tau-OS";
+    repo = "tau-helium";
+    rev = finalAttrs.version;
+    sha256 = "sha256-aeCcxg5CJ4gvRQoQm6gBgnSCyixNjCn9pirGJOOQCdI=";
```
Comment (Lord-Valen) [inline pkgs/data/themes/tau-helium/default.nix:18]:
```suggestion
    hash = "sha256-aeCcxg5CJ4gvRQoQm6gBgnSCyixNjCn9pirGJOOQCdI=";
```

File: pkgs/data/themes/tau-helium/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  lib,
+  fetchFromGitHub,
+  meson,
+  dart-sass,
+  ninja
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tau-helium";
+  version = "1.5.5";
```
Comment (Lord-Valen) [inline pkgs/data/themes/tau-helium/default.nix:12]: Latest is 1.5.8 if you wanted to bump it now.

Comment (Ashvith10) [reply pkgs/data/themes/tau-helium/default.nix:12]: @Lord-Valen ok, will bump the version

Comment (Lord-Valen) [reply pkgs/data/themes/tau-helium/default.nix]: Don't forget to change the commit message and PR title. :yum:

**Comment by Lord-Valen** [comment]
Everything looks good, let's just update the commit message/PR title to reflect the packaged version.

**Comment by Ashvith10** [comment]
How about now, @Lord-Valen ? I think I've covered it all.

**Review by Lord-Valen — APPROVED** [review]
LGTM

**Comment by lilyinstarlight** [comment]
@ofborg eval

Apologies for taking so long to get back to this!

Ofborg is a little stale, so i'm going to request a fresh eval, but i'm thinking this looks fine for merge otherwise (there is 1.5.9 that got released two days ago though...)

**Pull request closed without merging**